src/plots/Tensor/Tensor.xml

```diff
@@ -1,43 +1,83 @@
 <?xml version="1.0"?>
-  <Plugin name="Tensor" type="plot" label="Tensor" version="1.0" vartype="tensor,symmetrictensor" iconFile="Tensor.xpm">
-    <Attribute name="TensorAttributes" purpose="Attributes for the tensor plot" persistent="true" codefile="TensorAttributes.code">
-      <Field name="useStride" label="Use stride" type="bool">
-        false
+  <Plugin name="Tensor" type="plot" label="Tensor" version="1.0" enabled="true" mdspecificcode="false" engspecificcode="false" onlyengine="false" noengine="false" vartype="tensor" iconFile="Tensor.xpm">
```

Comment:
Why did you remove 'symmetrictensor' vartype?

Reply:
My input here is probably not too relevant to @ARSanderson changes but #4096 discusses issues with symmetric tensor support in VisIt

Reply:
Error on my part as I was pulling from the Vector.xml so to make sure they were similar. I missed that bit. Thanks for catching it.